### ISSUE TITLE
Google myBusiness integration and/or Gutenberg Block

### ISSUE THREAD

**Issue by SchneiderSam** [sybrew/the-seo-framework#459]
Interesting would be the possibility of Google myBusiness to display ratings. Keywords: RichSnippet and/or also as single Gutenberg Block.

**Comment by sybrew** [comment]
I don't think this should be a part of The SEO Framework; it doesn't contribute to ranking or search engine display. From what I gather, we only connect to [Google My Business' API](https://developers.google.com/my-business/), and display the information retrieved?

Do you have more in-depth information on this?

**Comment by SchneiderSam** [comment]
Ingenious that you are so open and detailed ;-)

> I don't think this should be a part of The SEO Framework; it doesn't contribute to ranking or search engine display.

Well, I don't see it that way. This article is interesting: https://backlinko.com/hub/seo/snippets
So this clearly has something to do with the CTR and the organic ranking.

So if I can display my Google My Business asterisks in the SERPs, then this is an optimization and it sets me apart from my SERP competitors.

Some inspirations:

- https://wordpress.org/plugins/google-places-reviews/
- https://www.seopress.org/features/google-structured-data-types/
- https://wordpress.org/plugins/schema-and-structured-data-for-wp/

And for the widget:

- https://wordpress.org/plugins/widget-google-reviews/
- https://wordpress.org/plugins/wp-google-places-review-slider/

Does that help?

**Comment by sybrew** [comment]
Yes, it does! 😄 

I am a bit wary about the fact that Google requires you to display the relayed information on your site. This is where the Gutenberg block comes in you suggested. It'll be a prominent requirement.

With that, I'll mark this for "maybe later," because I have yet to dive into creating blocks. Gutenberg's API is all over the place as well (for example, they want to remove lodash ([source](https://make.wordpress.org/core/2019/08/13/javascript-chat-summary-august-13-2019/))). 

Lastly, I don't have the time (yet) to maintain these. I'm already working 80+ hours a week 😅, and I hope the release of TSF 4.0 will shave off a dozen or so hours. That said, developers are free to add extensions. See https://tsf.fyi/contributing.
